Content.Server/Paint/PaintSystem.cs

```diff
@@ -0,0 +1,111 @@
+using Content.Shared.Popups;
+using System.Linq;
```

Comment:
.Linq
.Numerics
.Shared.Timing
.Shared.Humanoid
.Server.Decals

can be removed